client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -45,20 +66,24 @@ const StatsNavigation = ( props ) => {
 					{ sectionTitles.year }
 				</NavItem>
 			</NavTabs>
+			{ statsControl }
 			<FollowersCount />
 		</SectionNav>
 	);
 };
 
 StatsNavigation.propTypes = {
+	isJetpack: PropTypes.bool,
+	isWooConnect: PropTypes.bool,
 	section: PropTypes.string.isRequired,
 	slug: PropTypes.string,
+	siteId: PropTypes.number,
 };
 
 const connectComponent = connect(
-	state => {
+	( state, props ) => {
 		return {
-			slug: getSelectedSiteSlug( state )
```

Comment:
This is more a personal preference, so I wouldn't stop a 🚢  or anything, but I'd prefer we kept the slug in here instead of passing it through { ...props } from the parent.

Its one thing if we are creating a separation between presentation & container components, but since they are both `connect`'ed anyway I think it would make it easier to follow where the data is coming from.

My general thinking is that if a component is connected, then grab everything directly from the state tree that you can so that you don't need to be wondering what different pieces of data are.  If a component _isn't_ connected, then you will ideally only need to look at its connected parent to figure things out.